### PR TITLE
Update CASS Student Experience Page

### PR DESCRIPTION
**Pull request by kelnera**
The CASS Student Experience page needs to be changed

### CODE REVIEW

File: content/pages/student-experience/students.rst [hunk 1]
```diff
@@ -2,14 +2,42 @@ Student Experience
...
 
-Working at CASS provides students with an excellent opportunity to gain work
-experience. These are not your typical student positions. Students work closely
-with our customers and hosted clients to provide professional development and
-hosting services. Ever wanted to work closely with developers from some of the
-most prominent open source projects in the world? CASS provides a unique
-opportunity for students to do just that.
+Students at CASS work directly with a professional mento, and are engaged in all
+aspects of software engineering as well as DevOps operations, including managing
+and configuring servers and services for some of the world's most important open
```
Comment (alxngyn) [inline content/pages/student-experience/students.rst]: spelling: mentor

Also, I think this is a run-on sentence

File: content/pages/student-experience/students.rst [hunk 2]
```diff
@@ -2,14 +2,42 @@ Student Experience
...
+working with clients.
+
+* Students participate in all phases of a project: requirements gathering,
+  application design, development, testing, delivery, hosting and support of
+  industrial strength applications.
+* Professional staff provides ongoing mentoring and training in software
+  development, testing and system administration best practices.
+* Students interact directly with clients and develop an understanding and
+  appreciation for the business side of software engineering and application
+  hosting and support.
```
Comment (alxngyn) [inline content/pages/student-experience/students.rst]: This sentence is really hard for me to read. Maybe: 
```
Students will interact with clients to understand the business side of software engineering
```

**Comment by alxngyn** [comment]
aaaaaaaaaaa

File: content/pages/student-experience/students.rst (hunk 1 above)
Comment (kelnera) [inline content/pages/student-experience/students.rst]: spelling: lol

sentence: It's technically not a run-on, just long. Do you feel like it's too long? It's not one of my sentences and it's a little hard for me to figure out the structure, but if this is a general consensus then I'll take a closer look.

**Comment by kelnera** [comment]
@osu-cass/cass-media I've made a few changes, but before I had a chance to really think about this, I remembered that Carlos had already approved the current content.

The thing is @alxngyn has made a few really good points that drew my attention to the fact that this page is a hot mess that really needs to be fixed. Like, all of it. The important thing is to have some content there, but it would be nice to fix it now. I'm going to draft something and hopefully get it approved before this page goes live.

**Comment by kelnera** [comment]
@osu-cass/cass-media I made the changes anyway. I'm going to push to staging since there don't seem to be any issues, but if anyone has strong opinions about the wording, let me know.

**Pull request merged**